tools/src/main/java/org/apache/kafka/trogdor/common/WorkerUtils.java

```diff
@@ -248,9 +248,25 @@ static void createTopics(
     private static void verifyTopics(
         Logger log, AdminClient adminClient,
         Collection<String> topicsToVerify, Map<String, NewTopic> topicsInfo) throws Throwable {
-        DescribeTopicsResult topicsResult = adminClient.describeTopics(
-            topicsToVerify, new DescribeTopicsOptions().timeoutMs(ADMIN_REQUEST_TIMEOUT));
-        Map<String, TopicDescription> topicDescriptionMap = topicsResult.all().get();
+
+        Map<String, TopicDescription> topicDescriptionMap;
+        int retries = 0;
+        while (true) {
+            try {
+                DescribeTopicsResult topicsResult = adminClient.describeTopics(
+                        topicsToVerify, new DescribeTopicsOptions().timeoutMs(ADMIN_REQUEST_TIMEOUT));
+                topicDescriptionMap = topicsResult.all().get();
+                break;
```

Comment:
Suggest just using "return" here.  Then you can have the loop be a regular for loop.